src/py/mat3ra/made/tools/analyze.py

```diff
@@ -492,6 +492,8 @@ def get_local_extremum_atom_index(
     Returns:
         int: id of the atom with the minimum or maximum z-coordinate.
     """
+    if len(coordinate) == 2:
```

Comment:
This should be handled before invoking the function

Reply:
I'll use decorator